cmd/api-errors.go

```diff
@@ -806,6 +809,16 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Server side encryption specified but KMS authorization failed",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrNoAccessKey: {
+		Code:           "AccessDenied",
+		Description:    "No AWSAccessKey was presented",
+		HTTPStatusCode: http.StatusForbidden,
```

Comment:
Perhaps this message should reword AWSAccessKey to AccessKey, since the same message would be displayed for gateways

Reply:
I cannot this message is borrowed from S3 error response.